packages/eas-cli/src/commands/secret/create.ts

```diff
@@ -34,12 +34,16 @@ export default class EnvironmentSecretCreate extends Command {
     value: flags.string({
       description: 'Value of the secret',
     }),
+    json: flags.boolean({
```

Comment:
I'm not sure how useful that option would be, if there is some plain text output printed to the terminal.